src/proofgraph.h

```diff
@@ -0,0 +1,65 @@
+#ifndef PROOFGRAPH_H
+#define PROOFGRPAH_H
+
+
+#include <iostream>
+#include <set>
+
+#include "godot_cpp/classes/ref_counted.hpp"
+#include "godot_cpp/variant/vector3.hpp"
+#include "godot_cpp/classes/object.hpp"
+#include "godot_cpp/templates/hash_map.hpp"
+#include "godot_cpp/templates/hash_set.hpp"
+
+
+namespace godot{
```

Comment:
Why do we wrap this in Godot namespace? Presumably we don't want this to be considered part of the Godot standard library.

Reply:
It was in the godot documentation tutorial to do it that way and I was getting errors otherwise. Not sure if the errors were due to something else though

Reply:
leave it for now, I will look into it at a later date.